### ISSUE TITLE
0.38.1. Leaves swift (bridged) project with every pods' content undefined

### ISSUE THREAD

**Issue by adamjack** [CocoaPods/CocoaPods#3895]
Xcode Version 6.4 (6E35b)

Going from (I believe) 0.38 (working) to 0.38.1 (not working) I had to rename/change case of a number of the pods. Once past that I could not get the contents of the pods. Every reference would result in a "not defined" (sorry, don't have build log with errors.) It seems like nothing in the pods was available to my project. Restoring to 0.38 works.

Podfile:

```ruby
source 'https://github.com/CocoaPods/Specs.git'
platform :ios, '8.0'
xcodeproj 'SomeSwiftProject', 'ADHOC' => :release
#use_frameworks!

target "SomeTarget" do
        pod     'FBSDKCoreKit'
        pod     'FBSDKLoginKit'
        pod     'FBSDKShareKit'
        #pod    'SWRevealViewController'
        pod     'AFNetworking'
        pod     'AFNetworkActivityLogger'
        pod     'google-plus-ios-sdk'
        pod     'GoogleAnalytics'
        #pod    'google-api-client'
        pod     'Quantcast-Measure'
        pod     'Reachability'
        pod     'Google-Mobile-Ads-SDK'
        pod     'comScore-iOS-SDK'
        pod     'Toast'
        #pod 'Fabric'
        #pod 'Crashlytics'
        pod 'Lookback', :configurations => ['Debug']
        pod 'LookbackSafe', :configurations => ['Release']
        pod 'OpenSans'
end
target "SomeTargetTests" do
        pod     'OCMock'
        pod     'Google-Mobile-Ads-SDK'
end
```

**Comment by samjohn** [comment]
Also happening for me. Fix this please!!

**Comment by neonichu** [comment]
I have a hard time understanding the actual error without a build log, but I'm thinking this will be fixed by #3898 regardless.

**Comment by segiddins** [comment]
This should be fixed by 0.38.2.
